src/BTVNanoCommissioning/utils/histogrammer.py

```diff
@@ -137,11 +167,43 @@ def histogrammer(events, workflow):
             _hist_dict[f"{i}_dz"] = Hist.Hist(syst_axis, dz_axis, Hist.storage.Weight())
     elif "ttsemilep_sf" == workflow:
         obj_list = ["mu", "MET"]
+        obj_list.append("cjet")
+        _hist_dict["dr_cjet"] = Hist.Hist(
+            syst_axis, flav_axis, dr_axis, Hist.storage.Weight()
+        )
```

Comment:
this should only modified if the `if "c_ttsemilep_sf" == workflow: obj_list.append("cjet")`

Reply:
Removed cjet from simple ttsemilep